resources/helm/overlays/mesh-config/templates/enable-mesh-mtls.yaml

```diff
@@ -27,7 +27,7 @@ metadata:
     app: istio
     release: {{ .Release.Name }}
 spec:
-  host: "*.local"
+  host: "*.clusterDomain"
```

Comment:
No. This should be something like `*.{{ .Values.global.proxy.clusterDomain }}`.

Reply:
Made the change, both make gen and make lint passed on my side!